scripts/src/main/resources/scripts/command/docker

```diff
@@ -118,6 +122,26 @@ function doRun() {
   docker "${@}"
 }
 
+function doIsWsl1Used() {
+{
+  echo '@echo off'
+  echo 'ver > tmp-kernel-versiontest-result.txt'
+} > tmp-kernel-versiontest.bat
+
+cmd "/C tmp-kernel-versiontest.bat"
```

Comment:
We should stop doing such hacks.
Why creating a script containing one command instead of executing the command directly?
Why even writing a temporary file at all here?
Example:
```
echo $(wmic os get caption)
 Microsoft Windows 10 Pro
```
Or a generic way for calling via CMD:
```
$ MSYS_NO_PATHCONV=1 cmd /c ver

Microsoft Windows [Version 10.0.19044.2251]

hohwille@CE49454 MSYS /d/projects/mmm
$
```

BTW: I just had a similar discussion with @CREITZ25 in another PR review where a powershell script was created temporary and then called via powershell. The `docker` comandlet is full of such things and most probably none of them are required...